js/feature.js

```diff
@@ -0,0 +1,81 @@
+const speakers = [
+  {
+    name: 'Abdel Rahman',
+    image: './src-image/drzekri.jpg',
+    text: ' Vice President for Reasearch and Postgraduates Affairs',
+    description: 'ational Cancer Institute Verified email at nci.cu.edu.eg',
+  },
+  {
+    name: 'Mohamed Samy ',
+    image: './src-image/profa.jpg',
+    text: ' Cairo University Vice President for Community Service',
+    description: 'Doctor of Philosophy in Law from Faculty of Pharmacy',
+  },
+];
+
+speakers.forEach((speaker) => {
+  const div = document.querySelector('.speaker-info');
+  const dynamicContent = `
+       <card class="speaker ">
+                  <img src=${speaker.image} alt="Speaker" class="speaker-img">
+                  <div class="speaker-info">
+                    <h3 class="speaker-name">${speaker.name}
+                    </h3>
+                    <p class="speaker-desc">${speaker.text}
+                    </p>
+                    <p class="speaker-mor">${speaker.description}
+                    </p>
+                  </div>
+                </card>
+              </div>
+  `;
+  div.innerHTML += dynamicContent;
+});
+
+const speaker = [
+  {
+    image: './src-image/salma.jpg',
+    name: 'Salima Ikram',
+    text: 'Distinguished University Professor of Egyptology at The American University in Cairo',
+    description:
+      ' worked as an archaeologist in Turkey, Sudan, Greece and the United States',
+  },
+  {
+    image: './src-image/nagw.jpg',
+    name: 'Nagwa Sherif',
+    text: 'Architectural Engineering at The American University in Cairo',
+    description:
+      'She holds a PhD degree from University of Montreal, Canada in 1984',
+  },
+];
+speaker.forEach((speaker1) => {
+  const div = document.querySelector('#hidee');
+  const dynamicContent = `
+       <card class="speaker ">
+                  <img src=${speaker1.image} alt="Speaker" class="speaker-img">
+                  <div class="speaker-info">
+                    <h3 class="speaker-name">${speaker1.name}
+                    </h3>
+                    <p class="speaker-desc">${speaker1.text}
+                    </p>
+                    <p class="speaker-mor">${speaker1.description}
+                    </p>
+                  </div>
+                </card>
+              </div>
+  `;
+  div.innerHTML += dynamicContent;
+});
```

Comment:
- [ ] There should be six(6) speakers instead of four(4). Let's add the remaining speakers so as to match the design template.

- [ ] Currently, the items are overlapping when you switch from mobile to desktop at `width: 768`. Let's fix this overlapping by giving each individual item proper width, height, and padding.

Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/KTp6vMU.png) | ![](https://i.imgur.com/EqwmNns.png)